code/datums/materials/alloys.dm

```diff
@@ -0,0 +1,124 @@
+	color = "#585658"
+	value_per_unit = 0.225
+	strength_modifier = 1.3
+	integrity_modifier = 1.5
+	armor_modifiers = list(MELEE = 1.2, BULLET = 1.2, LASER = 1.5, ENERGY = 1.5, BOMB = 1.3, BIO = 1, RAD = 1.3, FIRE = 2, ACID = 1.5)
+	sheet_type = /obj/item/stack/sheet/mineral/plastitanium
+	categories = list(MAT_CATEGORY_RIGID=TRUE, MAT_CATEGORY_BASE_RECIPES=TRUE)
+	composition = list(/datum/material/titanium=1, /datum/material/plasma=1)
+
+/** Plasmaglass
+  *
+  * An alloy of silicate and plasma.
+  */
+/datum/material/alloy/plasmaglass
+	name = "plasmaglass"
+	desc = "Plasma-infused silicate. It is much more durable and heat resistant than either of its component materials."
+	color = "#dc90eb"
+	alpha = 210
+	strength_modifier = 1.2
+	integrity_modifier = 1.2
+	armor_modifiers = list(MELEE = 1.2, BULLET = 1.2, LASER = 1.5, ENERGY = 1.5, BOMB = 1.2, BIO = 1.3, RAD = 1.2, FIRE = 2, ACID = 3)
+	sheet_type = /obj/item/stack/sheet/plasmaglass
+	shard_type = /obj/item/shard/plasma
+	value_per_unit = 0.075
+	categories = list(MAT_CATEGORY_RIGID=TRUE, MAT_CATEGORY_BASE_RECIPES=TRUE)
+	composition = list(/datum/material/glass=1, /datum/material/plasma=0.5)
+
+/** Titaniumglass
+  *
+  * An alloy of glass and titanium.
+  */
+/datum/material/alloy/titaniumglass
+	name = "titanium glass"
+	desc = "A specialized silicate-titanium alloy that is commonly used in shuttle windows."
+	color = "#333135"
+	alpha = 210
+	strength_modifier = 1.2
+	integrity_modifier = 1.3
+	armor_modifiers = list(MELEE = 1.5, BULLET = 1.5, LASER = 1.2, ENERGY = 1.2, BOMB = 1.4, BIO = 1.3, RAD = 1.2, FIRE = 1.2, ACID = 2)
+	sheet_type = /obj/item/stack/sheet/titaniumglass
+	shard_type = /obj/item/shard
+	value_per_unit = 0.04
+	categories = list(MAT_CATEGORY_RIGID=TRUE, MAT_CATEGORY_BASE_RECIPES=TRUE)
+	composition = list(/datum/material/glass=1, /datum/material/titanium=0.5)
+
+/** Plastitanium Glass
+  *
+  * An alloy of plastitanium and glass.
+  */
+/datum/material/alloy/plastitaniumglass
+	name = "plastitanium glass"
+	desc = "A specialized silicate-plastitanium alloy."
+	color = "#232127"
+	alpha = 210
+	strength_modifier = 1.2
+	integrity_modifier = 1.3
+	armor_modifiers = list(MELEE = 1.4, BULLET = 1.4, LASER = 1.4, ENERGY = 1.4, BOMB = 1.4, BIO = 1.3, RAD = 1.2, FIRE = 2, ACID = 2.5)
+	sheet_type = /obj/item/stack/sheet/plastitaniumglass
+	shard_type = /obj/item/shard/plasma
+	value_per_unit = 0.125
+	categories = list(MAT_CATEGORY_RIGID=TRUE, MAT_CATEGORY_BASE_RECIPES=TRUE)
+	composition = list(/datum/material/glass=1, /datum/material/alloy/plastitanium=0.5)
+
+/** Alien Alloy
+  *
+  * Densified plasteel.
+  */
+/datum/material/alloy/alien
+	name = "alien alloy"
+	desc = "An extremely dense alloy similar to plasteel in composition. It requires exotic metallurgical processes to create."
+	color = "#313a68"
+	strength_modifier = 1.8 // It's twice the density of plasteel and just as durable. Getting hit with it is going to HURT.
```

Comment:
an alien alloy cleric mace (a one-handed weapon) would have a base force of 25, greater than that of a two-handed fire axe. again, that's probably okay, seeing as how you'd have to raid a specific space ruin AND get alien alloy to make such a mace, but I figured that I should point that out.

Reply:
I've nerfed this to parity with adamantine, it should be a bit more reasonable now.